app/views/mucr_options.scala.html

```diff
@@ -16,42 +16,72 @@
 
 @import forms.MucrOptions
 @import MucrOptions._
-@import views.components.fields.RadioOption
-@import views.Title
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import config.AppConfig
 
-@this(main_template: views.html.main_template)
+@this(  govukLayout: views.html.gds_main_template,
+govukButton: GovukButton,
+govukRadios: GovukRadios,
+govukInput: GovukInput,
+errorSummary: components.errorSummary,
+formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+appConfig: AppConfig)
 
 @(form: Form[MucrOptions])(implicit request: Request[_], messages: Messages)
 
-@main_template(title = Title("mucrOptions.title", None)) {
-    @helper.form(consolidations.routes.MucrOptionsController.save(), 'autoComplete -> "off") {
-        @helper.CSRF.formField
+@govukLayout(
+    title = messages("mucrOptions.title"),
+    backButtonTitle = messages("site.back"),
+    backButtonUrl = controllers.routes.ChoiceController.displayChoiceForm().url) {
 
-        @components.back_link(routes.ChoiceController.displayChoiceForm())
+    @formHelper(action = consolidations.routes.MucrOptionsController.save(), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
 
-        @components.error_summary(form.errors)
+        <span class="govuk-caption-xl">@messages("associate.heading", "")</span>
```

Comment:
Why there is empty String passing to the messages?

If the message patter doesn't suit for some specific situation I'll create a new message for this page. In general reusing messages is not a good practice, because if you change something you'll change in few places. I think every page should have his own messages keys

Reply:
done